service/dingding/dingding_test.go

```diff
@@ -0,0 +1,30 @@
+package dingding
+
+import (
+	"context"
+	"testing"
+)
+
+func TestService_Send(t *testing.T) {
+	type args struct {
+		subject string
+		content string
+	}
+
+	cfg := Config{
+		Token:  "xxx",
+		Secret: "xxx",
+	}
+
+	arg := args{
+		subject: "title",
+		content: "test content abcd",
+	}
+	s := New(&cfg)
+
+	t.Run("send", func(t *testing.T) {
+		if err := s.Send(context.Background(), arg.subject, arg.content); err != nil {
+			t.Errorf("Send() error = %v, wantErr %v", err, true)
+		}
+	})
+}
```

Comment:
I appreciate you adding tests but as discussed [here](https://github.com/nikoksr/notify/issues/1#issuecomment-783445488) and as you can see in services like `plivo` and `whatsapp` we are using [vektra/mockery](https://github.com/vektra/mockery) for generating mocks and writing tests based on those.

For an example on this, please take a look [here](https://github.com/nikoksr/notify/tree/main/service/plivo).

Reply:
I am not use Interface in dingding, the vektra/mockery is not suitable for this.